frontend/src/locales/index.tsx

```diff
@@ -1,15 +1,13 @@
-import { setupI18n } from '@lingui/core'
-import { Catalogs, I18n } from '@lingui/core/i18n'
+import { i18n } from '@lingui/core'
 
 /**
  * To create a new language or translation, for example, `en-custom`:
- * `npm run add-locale en-custom`
- * then import it here
- *  */
+ * Add it to the `locales` array in `.linguirc` and run `npm run extract`
+ */
 
 import catalogEn from './en/messages.js'
-const catalogs: Catalogs = { en: catalogEn }
-export const i18n: I18n = setupI18n({
-  language: 'en', // language to display for the site. Currently this is hard-coded and cannot be changed via UI.
-  catalogs,
-})
+const catalogs: Record<string, Record<string, string>> = {
+  en: catalogEn.messages,
+}
+i18n.load(catalogs)
```

Comment:
```suggestion
i18n.load('en', catalogEn.messages)
```

How about this as suggested by lingui's [migration guide](https://lingui.js.org/releases/migration-3.html#i18n-load-loads-a-catalog-for-a-single-locale)? Though I think it doesn't really make a difference given we only have one language.

Reply:
I used `i18n.load(catalogs)` in case there were ever other languages since I reckon it'd be cleaner that way. It doesn't matter though so I'll change it.